pybamm/models/submodels/thermal/x_lumped/x_lumped_2D_current_collectors.py

```diff
@@ -23,11 +24,11 @@ def set_rhs(self, variables):
         # TODO: update to allow different cooling conditions at the tabs
         self.rhs = {
             T_av: (
-                pybamm.laplacian(T_av)
-                + self.param.B * pybamm.source(Q_av, T_av)
-                + cooling_coeff * pybamm.source(T_av, T_av)
+                pybamm.laplacian(T_av - T_amb)
+                + self.param.B * pybamm.source(Q_av, T_av - T_amb)
```

Comment:
the second argument in `pybamm.source` is just the variable for which this is the equation for and the first argument is the source term in the equation (so that boundary conditions are handled correctly in finite elements). sorry, I don't think that is particularly well documented. As a result this should just be `pybamm.source(Q_av, T_av)`

Reply:
same for laplacian in other files

Reply:
ohh i see, thanks for the clarification